src/botPage/bot/tools.js

```diff
@@ -151,6 +151,8 @@ export const createDetails = contract => {
         +contract.exit_tick,
         +(contract.barrier ? contract.barrier : 0),
         result,
+        parseFloat(contract.entry_tick).toFixed(pipSize),
```

Comment:
You can use the plus operator (`+`) to convert the `contract.entry_tick` to a (float) number, this makes it consistent with the code found above.

Reply:
so should be somethign like `(+contract.entry_ticket).toFixed(pipSize)`?